core/web/router.go

```diff
@@ -80,6 +94,8 @@ func Router(app services.Application) *gin.Engine {
 		secureMiddleware(config),
 	)
 
+	engine.Use(explorerStatus(app))
```

Comment:
Let's place this middleware exclusively into the `api` so we don't set explorer status on static assets. We can simply place it underneath line 102 in the `api := engine.Group(...)` clause.